tutorials/3d/high_dynamic_range.rst

```diff
@@ -11,15 +11,15 @@ looks at their awesome looking model in the 3D DCC and says "looks
 fantastic, ready for integration!" then goes into the game, lighting is
 setup and the game runs.
 
-So where does all this HDR stuff thing come from? The idea is that
+So at point does all this HDR business come into play? The idea is that
```

Comment:
"at *what* point"

Reply:
> "at _what_ point" (or "_which_"? :D anyway, one word is missing)

@akien-mga :laughing: There you go again, cleaning up after me. Thanks!